src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  border: 1px solid $card-border-color;
+  border-radius: $card-border-radius;
+  @include box-sizing-border-box;
+
+  width: $card-width;
+  padding: $card-padding;
+  line-height: 18px;
+  color: $main-text-color;
+
+  @include transition(transform);
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
```

Comment:
change the card title text color to #34568b when the card is hovered (.card:hover .card__title)